### ISSUE TITLE
How to create an embedding matrix from embeddings generated by WordEmbeddingsModel

### ISSUE THREAD

**Issue by aidahalitaj** [JohnSnowLabs/spark-nlp#13541]
### Description

I neeed to combine spark-nlp with a model i keras&tensorflow. I have to create the embedding matrix to feed the Embedding layer of RNN model. But, I am struggeling to get words/vocabulary and the vectors from embeddings. 

```
# define sparknlp pipeline
document = DocumentAssembler()\
    .setInputCol("text")\
    .setOutputCol("document")

tokenizer = Tokenizer() \
    .setInputCols(["document"]) \
    .setOutputCol("token")

embeddings = WordEmbeddingsModel\
    .pretrained("w2v_cc_300d","sq")\
    .setInputCols(["document", "token"])\
    .setOutputCol("embeddings")

embeddingsFinisher = EmbeddingsFinisher()\
  .setInputCols("embeddings")\
  .setOutputCols("finished_embeddings")\
  .setOutputAsVector(True)
pipeline = Pipeline(stages=[document, tokenizer, embeddings, embeddingsFinisher])

# fit the pipeline to the training data
model = pipeline.fit(spark_train_df)

# apply the pipeline to the training data
result = model.transform(spark_train_df)
```

How to proceed further to create the matrix?

### Preferred Solution

I would like to have a guide on how to get the vectors and words from embedding. I saw that in older version there was a method getVectors() but that doesn't work with Spark NLP Version : 4.2.8.

### Additional Context
<img width="953" alt="image" src="https://user-images.githubusercontent.com/12566084/219908713-eef52659-706e-42da-8f22-ab19adb16f95.png">

`+--------------------+-----+--------------------+--------------------+--------------------+--------------------+
|                text|label|            document|               token|          embeddings| finished_embeddings|
+--------------------+-----+--------------------+--------------------+--------------------+--------------------+
|Joy Adowaa Buolam...|    0|[{document, 0, 13...|[{token, 0, 2, Jo...|[{word_embeddings...|[[0.0802000015974...|
+--------------------+-----+--------------------+--------------------+--------------------+--------------------+`
